site/ru/tutorials/images/classification.ipynb

```diff
@@ -0,0 +1,1359 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "OC8fIsalVrXd"
+      },
+      "source": [
+        "Примените все предыдущие дополнения. Здесь вы применяете к тренировочным изображениям масштабирование, поворот на 45 градусов, сдвиг ширины, сдвиг высоты, горизонтальное отражение и увеличение масштаба."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "gnr2xujaVrXe"
+      },
+      "outputs": [],
+      "source": [
+        "image_gen_train = ImageDataGenerator(\n",
+        "                    rescale=1./255,\n",
+        "                    rotation_range=45,\n",
+        "                    width_shift_range=.15,\n",
+        "                    height_shift_range=.15,\n",
+        "                    horizontal_flip=True,\n",
+        "                    zoom_range=0.5\n",
+        "                    )"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "K0Efxy7EVrXh"
+      },
+      "outputs": [],
+      "source": [
+        "train_data_gen = image_gen_train.flow_from_directory(batch_size=batch_size,\n",
+        "                                                     directory=train_dir,\n",
+        "                                                     shuffle=True,\n",
+        "                                                     target_size=(IMG_HEIGHT, IMG_WIDTH),\n",
+        "                                                     class_mode='binary')"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "AW-pV5awVrXl"
+      },
+      "source": [
+        "Визуализируем, как одно изображение будет выглядеть пять раз при случайной передаче этих дополнений к датасету.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "z2m68eMhVrXm"
+      },
+      "outputs": [],
+      "source": [
+        "augmented_images = [train_data_gen[0][0][0] for i in range(5)]\n",
+        "plotImages(augmented_images)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "J8cUd7FXVrXq"
+      },
+      "source": [
+        "### Создание генератора для проверочных данных\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "a99fDBt7VrXr"
+      },
+      "source": [
+        "Как правило, дополнение данных применяется только к тренировочным данным. В данном случае измените только масштаб проверочных изображений и преобразуйте их в batches, используя ImageDataGenerator.\n",
+        "\n",
```

Comment:
```suggestion
        "\n"
```
I *think* this may be the JSON error (tested in https://jsonlint.com/)
JSON can't have trailing commas 🙄 Try that